.github/workflows/manage-visual-tests.yml

```diff
@@ -0,0 +1,297 @@
+name: Manage Visual Tests
+
+concurrency:
+  group: visual-tests-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types: [synchronize, opened, reopened, closed, labeled, unlabeled]
+
+env:
+  VISUAL_TESTS_REPORTS_DIR: 'visual-tests-reports'
+  VISUAL_TESTS_RAW_REPORT_NAME: 'visual-tests-results.json'
+  VISUAL_TESTS_FINAL_REPORT_NAME: 'visual-tests-report.json'
+  VISUAL_TESTS_REPORT_COMMENT_ID: 'visual-changes-report-comment-${{ github.event.pull_request.number }}'
+  VISUAL_TESTS_CELLAR_BUCKET_NAME: ${{ vars.VISUAL_TESTS_CELLAR_BUCKET_NAME }}
```

Comment:
Using a repository variable here allowed me to run the action on my other repository, on another bucket from the ACME FOO org without having to modify the source code of the workflow :innocent: